draft-dcook-ppm-dap-interop-test-design.md

```diff
@@ -232,6 +232,7 @@ Upon receiving this command, the collector will poll the leader’s collect URL
 |Key|Value|
 |`status`|Either `"complete"` if the result was returned, `"in progress"` if the result was not yet ready, or `"error"` if an error occurred.|
 |`error` (optional)|An optional error message, to assist in troubleshooting. This will be included in the test runner logs.|
+|`report_count` (if complete)|A number, reflecting the count of client reports included in this aggregated result.|
```

Comment:
I thought about representing this as a string, but I suspect no test case will ever include nearly enough reports to exhaust the size of a JSON number.

Reply:
I agree on using a number over a string.

The attribute's name should be in camel case for consistency.

```suggestion
|`reportCount` (if complete)|A number, reflecting the count of client reports included in this aggregated result.|
```